modules/alert/mail/main.c

```diff
@@ -0,0 +1,197 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <haka/alert_module.h>
+#include <haka/log.h>
+#include <haka/error.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <curl/curl.h>
+
+#define FROM     "<haka@alert.com>"
```

Comment:
Maybe we should set it in configuration.
Set a default to alert.com could be annoying for alert.com owner. A reasonable default could be to use local hostname.